libsodium-sys/build.rs

```diff
@@ -4,31 +4,25 @@ extern crate pkg_config;
 use std::env;
 use std::path::PathBuf;
 
+#[cfg(target_env = "msvc")]
+extern crate vcpkg;
+
 fn main() {
     println!("cargo:rerun-if-env-changed=SODIUM_LIB_DIR");
     println!("cargo:rerun-if-env-changed=SODIUM_INC_DIR");
     println!("cargo:rerun-if-env-changed=SODIUM_STATIC");
 
-    if let Ok(lib_dir) = env::var("SODIUM_LIB_DIR") {
-        println!("cargo:rustc-link-search=native={}", lib_dir);
-
-        let mode = match env::var_os("SODIUM_STATIC") {
-            Some(_) => "static",
-            None => "dylib",
-        };
-        if cfg!(target_env = "msvc") {
-            println!("cargo:rustc-link-lib={0}=libsodium", mode);
-        } else {
-            println!("cargo:rustc-link-lib={0}=sodium", mode);
-        }
+    let mode = match env::var_os("SODIUM_DYNAMIC") {
```

Comment:
Why did you change `SODIUM_STATIC` to `SODIUM_DYNAMIC`?

Reply:
I forgot to change it back actually.